x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/CompositeBlock.java

```diff
@@ -86,22 +86,30 @@ public int getPositionCount() {
 
     @Override
     public int getTotalValueCount() {
-        throw new UnsupportedOperationException("Composite block");
+        // TODO: this works for aggregate metric double fields, because the four blocks are guarenteed to have the same length / positions.
```

Comment:
I think the `TODO:` part can be removed from these comments?  Since like the comment suggests for aggregate double metric field this works out.